test/compilable/extra-files/vcg-ast.d.cg

```diff
@@ -140,10 +140,104 @@ values!(__c_wchar_t)
 	pure nothrow @safe __c_wchar_t[] values()
 	{
 		__c_wchar_t[] values = null;
-		values ~= cast(__c_wchar_t)'\uffff';
```

Comment:
Just get rid of `~=` in the initial test. The [PR that introduced](https://github.com/dlang/dmd/pull/10155) this fixed a bug report that can be reproduced without the cat assign operator. What is problematic is the `cast(__c_wchar_t)` that should still exist.

This will make it so you don't need to put generated code for _d_arrayappendcTXImpl in the test.